lib/BloqadeLattices/src/tiles.jl

```diff
@@ -0,0 +1,39 @@
+abstract type Supercell{D} end
```

Comment:
```suggestion
abstract type AbstractSuperCell{D} end
```
CamelCase as well as adding "Abstract" for abstract types